src/oneD/StFlow.cpp

```
@@ -960,6 +1039,18 @@ void StFlow::setMeta(const AnyMap& state)
         m_zfixed = state["fixed-point"]["location"].asDouble();
         m_tfixed = state["fixed-point"]["temperature"].asDouble();
     }
+
+    // Two-point control meta data
+    if (state.hasKey("point-control")) {
+        const AnyMap& pc = state["point-control"].as<AnyMap>();
+        if (pc["type"] == "two-point") {
+            m_twoPointControl = true;
+            m_zLeft = pc["left-location"].asDouble();
+            m_zRight = pc["right-location"].asDouble();
+            m_tLeft = pc["left-temperature"].asDouble();
+            m_tRight = pc["right-temperature"].asDouble();
+        }
```

For future-proofing, I'd suggest a warning (use `warn_user(method, message, args to interpolate...)`) in case the type is not `two-point` rather than silently ignoring it.